recipes/inotify_simple/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "inotify_simple" %}
+{% set version = "1.1.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: fc2c10dd73278a1027d0663f2db51240af5946390f363a154361406ebdddd8dd
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [ win ]
+  
+requirements:
+  host:
+    - pip
+    - python
+    - enum34  # [ py<34 ]
+    
+  run:
+    - python
```

Comment:
Is `enum34` no run requirement?

Reply:
yes it should be, thank you.. should I also remove spaces on the selector there as well?

Reply:
Yes, please at all occurrences